cosmos/operators/_asynchronous/bigquery.py

```diff
@@ -110,6 +107,10 @@ def __init__(
         self.compiled_sql = ""
         self.gcp_project = ""
         self.dataset = ""
+        DbtRunAirflowAsyncBigqueryOperator.__bases__ = (
```

Comment:
It would be great if we added a comment on why we're redefining `__bases__` twice, potentially with some example of error message that lead to this solution

Reply:
yes, added comments to explain those. please check.